app/main/forms/auth_forms.py

```diff
@@ -62,6 +62,15 @@ class CreateUserForm(Form):
         ]
     )
 
+    phone_number = StringField(
+        'Phone number', id="input_phone_number",
+        validators=[
+            Regexp("^$|^\\+?([\\d\\s()-]){6,20}$",
+                   message="Phone numbers must be between 6 and 20 characters."
```

Comment:
I reckon fix all three messages now. 

Our phone number validation is a bit funky, saying 21 characters still wouldn't be true because it depends on what characters they are and in what order. We don't want to get into the details in our messaging though.